### PR TITLE
fix: update azure-kv plugin version to v1.0.1 & version matching

### PR DESCRIPTION
**Pull request by JeyJeyGao**
Feat:
- added `Plugin Version` input field

Fix:
- added azure-kv plugin v1.0.1 support
- updated the version matching function

Signed-off-by: Junjie Gao <junjiegao@microsoft.com>

### CODE REVIEW

File: NotationV0/data/azure_kv_versions.json
Comment (shizhMSFT) [inline NotationV0/data/azure_kv_versions.json:1]: Instead of changing the version from v1.0.0 to v1.0.1, please add a new version

**Review by shizhMSFT — CHANGES_REQUESTED** [review]
Also, we need to change the default version for installation

File: NotationV0/data/azure_kv_versions.json
Comment (JeyJeyGao) [reply NotationV0/data/azure_kv_versions.json:1]: It always installs the latest version of notation akv plugin, so there is no longer a need for v1.0.0.

Comment (JeyJeyGao) [reply NotationV0/data/azure_kv_versions.json:1]: User cannot choose AKV plugin version.

Comment (JeyJeyGao) [reply NotationV0/data/azure_kv_versions.json:1]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**